ofctrl/ofAction.go

```diff
@@ -57,6 +57,8 @@ const (
 	ActTypeSetField       = "setField"
 	ActTypeCopyField      = "copyField"
 	ActTypeMeter          = "meter"
+	ActTypeEncapsulate    = "encap"
```

Comment:
A similar question on the name to this one https://github.com/antrea-io/libOpenflow/pull/25#issuecomment-1216123654

Reply:
it is not only for nsh，it is for ethernet、nsh、mpls and mplstc